### PR TITLE
rbf fixes issues sum utxo and fee calculation

### PR DESCRIPTION
**Pull request by Serhii-Borodenko**
Issue Number (if Applicable): Fixes #

# Description

Please include a summary of the changes and which issue is fixed / feature is added. 

# Pull Request - Checklist  

- [x] Initial Manual Tests Passed
- [x] Double check modified code and verify it with the feature/task requirements
- [x] Format code
- [x] Look for code duplication
- [x] Clear naming for variables and methods

### CODE REVIEW

File: cw_bitcoin/lib/electrum_wallet.dart [hunk 1]
```diff
@@ -1362,26 +1363,11 @@ abstract class ElectrumWalletBase
...
-    if (transactionHex == null || transactionHex.isEmpty) {
-      return false;
-    }
-
-    return BtcTransaction.fromRaw(transactionHex).canReplaceByFee;
+  Future<bool> canReplaceByFee(ElectrumTransactionInfo tx) async {
+    final bundle = await getTransactionExpanded(hash: tx.txHash);
+    _updateInputsAndOutputs(tx, bundle);
+    if (bundle.confirmations > 0) return false;
+    return bundle.originalTransaction.canReplaceByFee;
```
Comment (OmarHatem28) [inline cw_bitcoin/lib/electrum_wallet.dart:1370]: how would this handle if it's called on a transaction that's already replaced?

File: lib/bitcoin/cw_bitcoin.dart [hunk 2]
```diff
@@ -398,9 +398,10 @@ class CWBitcoin extends Bitcoin {
   }
 
   @override
-  Future<bool> canReplaceByFee(Object wallet, String transactionHash) async {
+  Future<bool> canReplaceByFee(Object wallet, Object transactionHash) async {
```
Comment (OmarHatem28) [inline lib/bitcoin/cw_bitcoin.dart:401]: **Minor**
misleading name

File: cw_bitcoin/lib/electrum_wallet.dart (hunk 1 above)
Comment (Serhii-Borodenko) [reply cw_bitcoin/lib/electrum_wallet.dart]: Added a try-catch block to handle this exception.

File: lib/bitcoin/cw_bitcoin.dart (hunk 2 above)
Comment (Serhii-Borodenko) [reply lib/bitcoin/cw_bitcoin.dart]: fixed

**Review by OmarHatem28 — APPROVED** [review]

**Pull request merged**